kafkaesk/app.py

```diff
@@ -508,7 +508,7 @@ async def stop(self) -> None:
 
             for task in self._subscription_consumers_tasks:
                 try:
-                    await task
+                    await asyncio.wait([task])
```

Comment:
just curious, what's the purpose/advantage of a for-loop vs. calling `asyncio.wait()` on all the tasks simultaneously?

Reply:
Error propagation. When the regular await, if an error happened during it, it would stop right there and through an exception.